src/main/scala/no/ndla/articleapi/service/converters/HTMLCleaner.scala

```diff
@@ -71,7 +71,7 @@ trait HTMLCleaner {
 
     private def getIngress(content: LanguageContent, element: Element): Option[LanguageIngress] = {
       content.ingress match {
-        case None => extractIngress(element).map(LanguageIngress(_, content.language))
+        case None => extractIngress(element).map(LanguageIngress(_, None))
```

Comment:
Det ser riktig ut for meg. Når denne konverteres til en `domain.ArticleIntroduction` brukes språket fra `LanguageIngress`